src/components/LeftNavBar.js

```diff
@@ -1,34 +1,38 @@
 import { BsFillPersonFill, TbTextDirectionRtl, TbLayoutDashboard,
   IoNotificationsSharp, MdBackupTable, GiBulletBill, SiVirtualbox, RxDashboard, ImEnter, AiOutlineForm,
 } from '../utils/icons';
+import "./LeftNavBar.css"
 
 export default function LeftNavBar() {
- let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600'
-
+ let navItemCSS = 'flex items-center py-3 hover:bg-neutral-900 rounded-lg active:bg-pink-600 mx-4'
+ let iconCSS = "p-1 text-3xl"
+ 
   return (
-    <div className="w-screen grid grid-cols-2 border-[2px] border-white ">
-      <div className="w-[180px] h-[calc(100%-4px)] fixed top-[2px] left-5 border-[2px] border-black bg-gradient-to-b from-neutral-800 to-neutral-950 rounded-lg">
-        <div className="mx-4 text-white text-sm">
-          <div className='flex items-center py-3 border-b-2 mb-2'><TbLayoutDashboard className="p-1" />Material Dashboard </div>
-          <div className="block h-[calc(50vh)] overflow-auto overscroll-y-auto">
+    <div className="w-screen grid grid-cols-2 border-[2px] border-white">
+      <div className="w-[250px] h-[calc(100%-40px)] fixed top-[2px] left-5 border-[2px] border-black bg-gradient-to-b from-neutral-800 to-neutral-950 rounded-lg align-tems">
```

Comment:
This is a comment for later on. @neli-rashkova and @HappyHanSolo one thing to think about is this.

If this w-[250px] is relevant for both of you guys (because Neli needs to push away by this same amount, and hanson need it because of how much width he needs for the left nav bar), then what happens when Hanson introduces some other changes and requires this width to change?

Instead of having to worry about all the manual places where w-[250px] was defined, this could be done on tailwind's config level, and then both people can use it, so that the width is not hard coded.